### PR TITLE
Reworks the Sulaco ruin

### PR DESCRIPTION
**Pull request by VileBeggar**
## About The Pull Request
**the original ruin:**
sulaco_original
**the slightly better version**
sulaco_new

updates the ruin's visuals, adds a pair of fluff papers to it, alongside some minor loot (very damaged tgmc gear for any cosplayers out there) and some gamer loot (rouny plushie). it also now has a GPS tracker, a generic Distant Signal like the rest of the space ruins

the armour is fairly weak protection wise. it has worse combat protection than sec armour, but it has very good biological and acid protection. it inherits some of the other qualities of marine armour (spaceproof, acidproof, some fire resistant as well). i wanted to toss a bone to all those lovely peeps who want to rep the marines, though i don't mind changing the values of it if need be. military jumpsuit, helmet and vest are all included in the ruin.
marined

also it's my first time mapping something, so i would _really_ love feedback on what to change if need be.

## Why It's Good For The Game
i like this ruin a lot, and it adds a good bit to the whole atmosphere of ss13. seeing two completely different servers connected in such a neat little way gave me a giggle the first time i discovered it. given all of that, i felt like it was a good candidate for sprucing up. i think updating the ruins bit by bit should help freshen up exploration, at least a little bit

## Changelog
:cl: Vile Beggar
add: The Derelict Sulaco ruin was spruced up with a new look and some neat little goodies (free rouny plush).  
/:cl:

### CODE REVIEW

File: code/modules/mapfluff/ruins/spaceruin_code/derelict_sulaco.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	desc = "A tactical black helmet, barely sealed from outside hazards with a plate of glass and not much else. Not as protective as it used to be, but it is still completely functional."
+	armor_type = /datum/armor/derelict_marine
+
+/datum/armor/derelict_marine
+	melee = 20
+	bullet = 20
+	laser = 10
+	energy = 10
+	bomb = 35
+	bio = 100
```
Comment (optimumtact) [inline code/modules/mapfluff/ruins/spaceruin_code/derelict_sulaco.dm:33]: why do people do this

Comment (optimumtact) [reply code/modules/mapfluff/ruins/spaceruin_code/derelict_sulaco.dm:33]: every goddamna rmour type

**Comment by YakumoChen** [comment]
bomb armour seems pretty high, not sure why it even needs much-if-any bomb armour at all

**Comment by Addust** [comment]
based

**Comment by Addust** [comment]
give the terminals some fluff text as well, preferably related to the consoles at the same location on the actual TGMC sulaco

**Review by Maurukas — CHANGES_REQUESTED** [review]
<details>
<summary> There's a uniform on the same tile as this R-wall </summary> 

image
</details>

Some lore in the terminals would be nice - but that can easily be done in a future PR by anyone who would care to add it. The terminals aren't powered by default, so it's just a nice to have.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**